types/bech32/bech32_test.go

```diff
@@ -0,0 +1,24 @@
+package bech32_test
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/libs/bech32"
```

Comment:
Just saw this @marbar3778 ! Is this expected? I think you wanted to test the cosmos-sdk version!

Reply:
https://github.com/cosmos/cosmos-sdk/pull/6201/files

Reply:
good catch